### ISSUE TITLE
Finetuning model on google Colab

### ISSUE THREAD

**Issue by Jines794** [MIC-DKFZ/nnUNet#1577]
Hello ! 
So, I have trained nnUNet on Colab(following this tutorial https://github.com/IML-DKFZ/nnunet-workshop/blob/main/nnU-Net_Workshop.ipynb ) up to 500 epoch with nnUNetTrainerDA5. 
I will like to perform validation instead of waiting for it to reach 1000 epoch. I know that I could fiddle with some hyperparameters in the nnUNetTrainer.py code , such as setting num.epochs to 500 instead of 1000 . the issue is that I don't have access to the code on Colab. 
Do anyone have a clue on how solve the issue ?  

Thanks

**Comment by ancestor-mithril** [comment]
The simplest solution is to monitor and stop the training manually when it reaches 500 epochs, then use `-chk checkpoint_latest.pth` or `-chk checkpoint_best.pth` to predict on the validation set. 
```
nnUNetv2_predict -i INPUT -o OUTPUT -d D -c C -device cuda -f 0 -chk checkpoint_latest.pth
```

**Comment by ancestor-mithril** [comment]
Or you can use `-tr nnUNetTrainer_250epochs` when training. 
Available trainers: 
https://github.com/MIC-DKFZ/nnUNet/blob/4612d35b1b80d558f3e1a650decd408f6f70c68a/nnunetv2/training/nnUNetTrainer/variants/training_length/nnUNetTrainer_Xepochs.py

**Comment by Jines794** [comment]
> The simplest solution is to monitor and stop the training manually when it reaches 500 epochs, then use `-chk checkpoint_latest.pth` or `-chk checkpoint_best.pth` to predict on the validation set.
> 
> ```
> nnUNetv2_predict -i INPUT -o OUTPUT -d D -c C -device cuda -f 0 -chk checkpoint_latest.pth
> ```

Thank you for your reply ! Can I used this command if I used the 5 split

**Comment by Jines794** [comment]
> Or you can use `-tr nnUNetTrainer_250epochs` when training.
> Available trainers:
> https://github.com/MIC-DKFZ/nnUNet/blob/4612d35b1b80d558f3e1a650decd408f6f70c68a/nnunetv2/training/nnUNetTrainer/variants/training_length/nnUNetTrainer_Xepochs.py

Thank you for your reply !

**Comment by Jines794** [comment]
> The simplest solution is to monitor and stop the training manually when it reaches 500 epochs, then use `-chk checkpoint_latest.pth` or `-chk checkpoint_best.pth` to predict on the validation set.
> 
> ```
> nnUNetv2_predict -i INPUT -o OUTPUT -d D -c C -device cuda -f 0 -chk checkpoint_latest.pth
> ```

For training, I used the 5 split cross validation, and only one split has been trained. how does it work to specify the input folder ?

**Comment by TueNguyen2006** [comment]
@Jines794 Hey would you provide your code on colab for me to refer. Thanks
